tests/docs-screenshots.spec.ts

```diff
@@ -68,43 +90,646 @@ test("instances", async ({ page }) => {
+    .getByRole("row", { name: "Exposed CPU limit" })
+    .getByRole("button")
+    .click();
+  await page.getByPlaceholder("Number of exposed cores").fill("4");
+  await page.getByPlaceholder("Enter value").fill("8");
+  await page
+    .getByRole("button", { name: "Create and start", exact: true })
+    .click();
+  await page.getByTestId("notification-close-button").click();
+
+  await visitInstance(page, vminstance);
+  await page.waitForSelector("text=General");
+  await page.getByTestId("tab-link-Console").click();
+  await page.waitForTimeout(45000);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/desktop_console.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  // Open File Explorer and navigate to /Documents
+  await page.mouse.click(300, 280, { delay: 1000 });
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowRight");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+
+  // Creates new folder
+  await page.mouse.click(725, 525, { button: "right", delay: 1000 });
+  await page.keyboard.press("ArrowDown");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("ArrowUp");
+  await page.waitForTimeout(1000);
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+  await page.keyboard.type("hello_world");
+  await page.keyboard.press("Enter");
+  await page.waitForTimeout(1000);
+
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/hello_world_desktop.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await forceStopInstance(page, vminstance);
+  await deleteInstance(page, vminstance);
+});
+
+test("LXD - UI Folder - Project", async ({ page }) => {
+  await page.setViewportSize({ width: 1440, height: 800 });
+  await gotoURL(page, "/ui/");
+
+  // Project Screenshots
+  await page.waitForTimeout(1000);
+  await page.getByRole("button", { name: "default", exact: true }).click();
+  await page.getByRole("button", { name: "Create project" }).click();
+  await page.waitForLoadState("networkidle");
+  await page.getByPlaceholder("Enter name").fill("my-project");
+  await page
+    .getByRole("combobox", { name: "Features" })
+    .selectOption("customised");
+  await page.getByRole("combobox", { name: "Features" }).focus();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/create_project.png",
+    clip: getClipPosition(240, 0, 1440, 960),
+  });
+
+  await page.getByPlaceholder("Enter name").fill("my-restricted-project");
+  await page.getByText("Allow custom restrictions on").click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/UI/create_restr_project1.png",
+    clip: getClipPosition(240, 0, 1440, 760),
+  });
```

Comment:
Features should be set to default lxd for this screenshot

<img width="1862" height="727" alt="image" src="https://github.com/user-attachments/assets/6b0cee7e-1d94-4022-82f8-a626dd0025f5" />